project/libs/blog/blog-like/src/blog-like-module/blog-like.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { BlogLikeRepository } from './blog-like.repository';
+import { BlogLikeEntity } from './blog-like.entity';
+
+@Injectable()
+export class BlogLikeService {
+  constructor(
+    private readonly blogLikeRepository: BlogLikeRepository
+  ) { }
+  public async create(postId: string, userId: string): Promise<BlogLikeEntity> {
+    const newLike = new BlogLikeEntity({ postId, userId })
+    await this.blogLikeRepository.save(newLike);
+    return newLike;
```

Comment:
Entity должен использоваться в Repository 

```suggestion
return await this.blogLikeRepository.create({postId, userId})
```

Reply:
Из-за разделения слоев? В учебном проекте так же было...